doc/config.rst

```diff
@@ -295,6 +295,14 @@ General configuration
 
    .. versionadded:: 1.3
 
+.. confval:: manpages_url_pattern
```

Comment:
I feel `manpages_url` is simpler. What do you think?

Reply:
I don't care - but i felt that "pattern" implied it was a bit more than just a prefix...

Reply:
If you don't opposite, could you rename this please?

Reply:
sure. done.